### PR TITLE
Add API

### PR DESCRIPTION
**Pull request by Pazzo97**
- Created a new game with the name of my choice by using the API.
- Implemented the "Refresh" button (receiving data from the API and parsing the JSON).
- Implemented the form "Submit" button (sending data to the API).
- Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,45 @@
...
+  };
+  postRecord(record).then(() => {
+    event.target.reset();
+    getScores();
+  });
+});
+
+refreshButton.addEventListener('click', () => {
+  getScores();
+});
```
Comment (sja-thedude) [inline src/index.js:46]: - [ ] Kindly make sure that when the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Pazzo97, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good PR Title and Description
✔️ Correct git-flow
✔️ Professional Readme
✔️ Great job so far

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -27,17 +27,18 @@ const postRecord = async (record) => {
...
   const record = {
     user: formData.get('user'),
     score: formData.get('score'),
   };
-  postRecord(record).then(() => {
-    event.target.reset();
-    getScores();
-  });
+  await postRecord(record);
+  // .then(() => {
```
Comment (zieeco) [inline src/index.js:38]: **[OPTIONAL]**

- [ ] Please, kindly remove the unused code, It makes your code not clean

File: src/index.js [hunk 3]
```diff
@@ -27,17 +27,18 @@ const postRecord = async (record) => {
...
+  event.target.reset();
+  //   getScores();
+  // });
```
Comment (zieeco) [inline src/index.js:41]: **[OPTIONAL]**

- [ ] Please, kindly remove the unused code, It makes your code not clea

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Pazzo97 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**